services/movie_session.py

```diff
@@ -42,3 +42,13 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
Consider specifying the types of keys and values in the dictionary returned by `get_taken_seats` for clarity. For example, use `list[dict[str, int]]`.